docs/_sass/minima/_layout.scss

```diff
@@ -166,7 +166,7 @@
 .post-content h1 {
   @include relative-font-size(2.625);
   letter-spacing: -1px;
-  line-height: 1.15;
+  line-height: 0;
```

Comment:
why did you change this to 0?

Reply:
Reduce the spacing between the title (guide) and the subtitle (version number) xD

Reply:
Ohh, ok nice